test/src/main/java/org/springframework/security/test/web/reactive/server/JwtMutator.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.test.web.reactive.server;
```

Comment:
It's surprising to see this class alongside `SecurityMockServerConfigurers`. Instead, let's keep this class inside `SecurityMockServerConfigurers` and consider altering that pattern in a separate ticket.

Let's do the same for `JwtRequestPostProcessor`.

Reply:
Done

I let you fill the ticket, recent experience proved I'm very bad at it.

Reply:
ha ha - honestly, I think you are very well-spoken

I need to improve there, too - tickets often a place where contributors want to start a discussion, not simply propose something and have it rejected out of hand.